Makefile

```diff
@@ -92,7 +87,7 @@ src/extensions/npm/extensions/dist: src/extensions/npm/extensions/node_modules
 	yarn compile:extension-types
 
 src/extensions/npm/extensions/node_modules: src/extensions/npm/extensions/package.json
-	cd src/extensions/npm/extensions/ && ../../../../node_modules/.bin/npm install --no-audit --no-fund
```

Comment:
We aren't really using the package-lock file anyway since we override it whenever we do a `make build-extension-types` invocation.

Reply:
Just wondering should we include package-locks... anyway that is a story for an another PR.

Reply:
Yeah probably. Once these are libraries and build themselves for the releases.